apps/exploits/tests/test_api.py

```diff
@@ -61,14 +61,16 @@ def load_data():
         ps_component="ps-component-2",
     )
     AffectFactory(
-        resolution=Affect.AffectResolution.WONTFIX,
```

Comment:
Is there any reason to change WONTFIX to DEFER here?

Reply:
Do you think there is no value to doing so? If not, I will remove. I figured it made sense to test a similar affect resolution status here.

Reply:
I mean I am not against testing DEFER too, it just no makes sense to me to stop testing WONTFIX when we keep using it. If you prefer testing DEFER, just introduce another test(s) or parametrize the existing

Reply:
Ah here I'm testing both

```py
    AffectFactory(
        resolution=Affect.AffectResolution.WONTFIX,
        affectedness=Affect.AffectAffectedness.AFFECTED,
        flaw=f2,
        ps_module="ps-module-2",
        ps_component="ps-component-2",
    )
    AffectFactory(
        resolution=Affect.AffectResolution.DEFER,
        affectedness=Affect.AffectAffectedness.AFFECTED,
        flaw=f3,
        ps_module="rhel-6",
        ps_component="firefox",
    )
    AffectFactory(
        resolution=Affect.AffectResolution.DEFER,
        affectedness=Affect.AffectAffectedness.AFFECTED,
        flaw=f4,
        ps_module="rhel-5",
        ps_component="kernel",
    )
```

Reply:
I had been testing a combination, but I don't think adding coverage for DEFER here adds any value, so I reverted it.